src/app/plugin/codec-simple/codec-simple.c

```diff
@@ -24,9 +24,12 @@
  */
 #include "chip-zcl-codec.h"
 
+#include <assert.h>
 #include <memory.h>
 #include <stdint.h>
 
+#include "../../gen/gen-command-id.h"
```

Comment:
other places in the tree will symlink to this header so that code doesn't have to know the tree structure

Reply:
Most things worked if I just used "gen/gen-command-id.h" here, but the "main build" did not.  If there's a way to make that work, I'd love to know how.